### PR TITLE
Novation Dicer: remove flanger mapping with quickeffect toggle

### LINKED ISSUE #13134: Novation Dicer default mapping
### Bug Description

Something feels off/wrong with the default Novation Dicer mapping. Get an error on Mixxx start about an LED issue. Some controls don't appear to do anything. I could probably come up with fixes for the non-working functions/buttons, but I don't think I have the knowledge to fix whatever's going on with the LED error.
```
MIDI output message 0x9F 0x3E has invalid MixxxControl [Channel2], flanger
MIDI output message 0x9C 0x3E has invalid MixxxControl [Channel1], flanger
```

### Version

2.4.0 HEAD

### OS

Windows 11

### PR DESCRIPTION
**Pull request by Swiftb0y**
Fixes #13134

Tbh. this is just an idea. I don't know how much sense it makes. I would appreciate alternative suggestions.

### CODE REVIEW

**Comment by Swiftb0y** [comment]
@Phr3d13 is this solution okay for you? @JoergAtGithub do you have time to review and merge this?

File: res/controllers/Novation Dicer.midi.xml [hunk 1]
```diff
@@ -274,7 +274,7 @@
             <control>
                 <status>0x9f</status>
                 <midino>0x3e</midino>
-                <group>[Channel2]</group>
+                <group>[QuickEffectRack1_[ChannelI]_Effect1]</group>
```
Comment (JoergAtGithub) [inline res/controllers/Novation Dicer.midi.xml:277]:
```suggestion
                <group>[QuickEffectRack1_[Channel2]_Effect1]</group>
```

File: res/controllers/Novation Dicer.midi.xml [hunk 2]
```diff
@@ -574,7 +574,7 @@
             <control>
                 <status>0x9c</status>
                 <midino>0x3e</midino>
-                <group>[Channel1]</group>
+                <group>[QuickEffectRack1_[ChannelI]_Effect1]</group>
```
Comment (JoergAtGithub) [inline res/controllers/Novation Dicer.midi.xml:577]:
```suggestion
                <group>[QuickEffectRack1_[Channel1]_Effect1]</group>
```

File: res/controllers/Novation Dicer.midi.xml [hunk 3]
```diff
@@ -574,7 +574,7 @@
...
                 <key>NovationDicer.flangeEffect</key>
```
Comment (JoergAtGithub) [inline res/controllers/Novation Dicer.midi.xml:578]:
```suggestion
                <key>NovationDicer.enableEffect</key>
```
The function name is not matching to the functionality anymore.

**Review by JoergAtGithub — CHANGES_REQUESTED** [review]

**Comment by Swiftb0y** [comment]
whoops, embarrasing, see fixup

**Review by JoergAtGithub — APPROVED** [review]
LGTM now!

**Comment by Swiftb0y** [comment]
Thank you for the review ;)

**Comment by ronso0** [comment]
> Also, if it's removed, why is it still an option?

This doesn't remove the Flanger effect (why?), only the deprecated direct effect control from the time when Mixxx only had a handful of effects, and only for this particular mapping.

**Pull request merged**